src/executor/admin/SubmitJobExecutor.cpp

```diff
@@ -18,20 +18,35 @@ folly::Future<Status> SubmitJobExecutor::execute() {
 
     auto *sjNode = asNode<SubmitJob>(node());
     auto jobOp = sjNode->jobOp();
-    meta::cpp2::AdminCmd cmd = meta::cpp2::AdminCmd::COMPACT;
+    std::vector<std::string> jobArguments;
+    auto spaceName = qctx()->rctx()->session()->space().name;
+    jobArguments.emplace_back(std::move(spaceName));
+
+    meta::cpp2::AdminCmd cmd;
     if (jobOp == meta::cpp2::AdminJobOp::ADD) {
-        std::vector<std::string> params;
-        folly::split(" ", sjNode->params().front(), params, true);
-        if (params.front() == "compact") {
+        auto& params = sjNode->params();
```

Comment:
if the parser directly to use `AdminCmd`, the converted code is not needed.

Reply:
maybe I  will fix it later BTW would support REBUILD sentence in V2.0 ?

Reply:
I think it's not needed, and we need to modify the doc.